README.md

```diff
@@ -1,2 +1,42 @@
-# persistent-volume-migrator
-Tool to migrate Kubernetes In-tree and Flex volumes to Ceph-CSI
+# What is persistent-volume-migrator?
+
+The persistent-volume-migrator tool aims to help migrate Flex RBD Volumes to [Ceph-CSI](https://github.com/ceph/ceph-csi) RBD Volumes.
+
+# Getting Started
+**NOTE**: The Migration procedure will come with a downtime; As we need to scale down the applications using the Volumes before migration.
+## Prerequisite
+This guide assumes you have created a Ceph cluster with Flex and [Ceph-CSI](https://github.com/ceph/ceph-csi).
+1. We assume that you have healthy cluster i.e. `ceph status` shows `health: OK`
+2. Minimum ceph version should be Ceph-CSI supported
+3. Stop the application pods that are consuming the flex volume(s) that need to be converted.
+4. Ceph-CSI storageClass should be created in which you want to migrate.
+
+## Steps to Migrate
+1. Stop the application pods that are consuming the flex volume(s) that need to be converted.
+2. Create rbac required to run migrator pod
+   1. `kubectl create -f tests/manifests/rbac.yaml`
```

Comment:
The yamls have been merged to migrator.yaml, right?

Reply:
yes...will update